BeVolt/Drivers/src/LTC6811.c

```diff
@@ -75,7 +75,185 @@ Copyright 2017 Linear Technology Corp. (LTC)
 /*********************************************************/
 /*** Code that was added by UTSVT. ***/
 /*********************************************************/
-void LTC6811_Init(cell_asic *battMod){	
+
+static void cs_set(uint8_t state){
+	if(state == 0){
+		GPIO_ResetBits(GPIOB, GPIO_Pin_6);
+	}else{
+		GPIO_SetBits(GPIOB, GPIO_Pin_6);
+	}
+}
+void delay_u(uint16_t micro){
+  uint32_t delay = SystemCoreClock / 1000000;
+	for(int i = 0; i < micro; i++){
+		for(int j = 0; j < delay; j++);
+	}
+}
+
+void delay_m(uint16_t milli){
+  uint32_t delay = SystemCoreClock / 1000;
+	for(int i = 0; i < milli; i++){
+		for(int j = 0; j < delay; j++);
+	}
+}
+
+/* wakeup_idle
+ * Wake isoSPI up from idle state
+ * @param number of ICs in the daisy chain
+ */
+void wakeup_idle(uint8_t total_ic){
+	volatile uint8_t tempReg;		// Temporary buffer
+  for (int i = 0; i<total_ic; i++){
+    cs_set(0);
+    //delayMicroseconds(2); //Guarantees the isoSPI will be in ready mode
```

Comment:
Why is this commented out? Is it necessary to have, or should it be removed entirely?